WORKSPACE

```diff
@@ -0,0 +1,142 @@
+workspace(name = "com_google_api_gapic_generator")
```

Comment:
Would it make sense to add a file comment with the date these were generated?

Reply:
This wasn't generated, it was handwritten. (so I don't think it makes sense...)